data/presets/highway/cycleway/traffic_island_shared.json

```diff
@@ -0,0 +1,36 @@
+{
+    "locationSet": {
+        "exclude": [
+            "fr",
+            "lt",
+            "pl",
+            "de",
+            "il",
+            "ps"
+        ]
+    },
+    "icon": "temaki-pedestrian_and_cyclist",
+    "geometry": [
+        "line"
+    ],
+    "tags": {
+        "cycleway": "traffic_island"
+    },
+    "addTags": {
+        "highway": "cycleway",
+        "cycleway": "traffic_island"
+    },
```

Comment:
**`addTags`**
I don't think that we need `addTags` because all that we need can be part of `tags`.
Docs: https://github.com/ideditor/schema-builder?tab=readme-ov-file#addtags

The `cycleway/bicycle_foot` preset uses `addTags` to force the `bicycle=designated` but I am not sure we need to force this, see #1411.

**`removeTags`**
The docs at https://wiki.openstreetmap.org/wiki/Tag%3Acycleway%3Dtraffic_island and https://wiki.openstreetmap.org/wiki/Tag:footway%3Dtraffic_island go into details on the `crossing:island`. My understand is, that it is likely that a way has a `crossing:island` when it is migrated to a `*=traffic_island` and in this case, we want to remove all `crossing:island` because those only belong on `crossing=*` ways.

```suggestion
    "removeTags": {
        "crossing:island": "*",
    },
```

Reply:
_[please ignore previous deleted comment 🤦]_

I think these `removeTags` belong on the crossing presets, not the traffic island presets right? because we want `crossing:*` to be removed when changing from crossing to traffic_island

Reply:
regarding `bicycle=designated` should we keep it for consistency and change it later if #1411 is accepted? 

Otherwise it might be confusing if only the refuge island preset matches, but not the connected paths. [Example here](https://pr-1068--ideditor-presets-preview.netlify.app/id/dist/#id=w1289722394).

Reply:
@k-yle I agree, lets keep the presets in sync and update them later, if needed.

Reply:
> [please ignore previous deleted comment 🤦]

JFY thanks for this, I lead the comments via mail and this helps because updates don't get emailed :)

Reply:
> I think these removeTags belong on the crossing presets, not the traffic island presets right? because we want crossing:* to be removed when changing from crossing to traffic_island

Let's leave this out of V1 of this addition and maybe add it later… It makes this PR more complex that it has to be.

The use case I was thinking of is… Users use `crossing:island=yes` on ways when they don't split of the `traffic_island` part of the way. Then, at some point, users want to add details and split off the `traffic_island` part. Which means that a ways that already has `crossing:island=yes` gets transformed into a `…=traffic_island` which would be when this `removeTags` gets triggered (right?).

But again, this should be done separately…